schema/config-linux.json

```diff
@@ -204,7 +204,7 @@
                                     "type": "object",
                                     "properties": {
                                         "pageSize": {
-                                            "type": "string"
+                                            "$ref": "defs.json#/definitions/stringPointer"
```

Comment:
This is a bit sticky, since the Markdown has [`pageSize` as required][1] but the Go [has it as a pointer][2].  I don't see the point of declaring `hugepageLimits` without setting both (in which case I don't think they should be pointers in either Go or JSON Schema), but I haven't read enough of the history to say for sure.

[1]: https://github.com/opencontainers/runtime-spec/blob/90451c4923edfa8162425707c8a496085277267e/config-linux.md#huge-page-limits
[2]: https://github.com/opencontainers/runtime-spec/blob/90451c4923edfa8162425707c8a496085277267e/specs-go/config.go#L198-L199